tox.ini

```diff
@@ -18,4 +18,5 @@ ignore =
     E123, E124, E126,
     E226,
     E402, E501, E722,
-    F812, F841, W503
+    F812, F841, W503,
+    W504
```

Comment:
For later reference; We ignored `W504` because the rule of `flake8==3.5.0` became the anti-pattern in `flake8==3.6.0`.

See https://github.com/LintlyCI/Flake8Rules/blob/ed09f9d/_rules/W503.md
> Note: Despite being in the best practice section, this will soon be considered an anti-pattern.